### PR TITLE
Business site - Vanessa

### PR DESCRIPTION
**Pull request by vanhaj**
Netlify link:[https://zen-aryabhata-996c71.netlify.app](url)

### CODE REVIEW

**Comment by HWallberg** [comment]
Hey Vanessa! Your netlify link doesn't seem to work :)

**Comment by vanhaj** [comment]
https://zen-aryabhata-996c71.netlify.app
[Netlify](https://zen-aryabhata-996c71.netlify.app)

File: code/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Business Site
+
+This weeks assignment was to to create a business one pager website. The focus was on the hero image/video, a sign up form and an accordion.
+
+## The problem
+
+I planned the website by using figma.
+If I had more time, I would change the FAQ and the styling around it. I would've wanted to have the questions stay in position when opening each one and adding some more styling. I also would've added a navbar on the top or some kind of logo.
```
Comment (HWallberg) [inline code/README.md:8]: Interesting to read your thoughts here! Well done

File: code/index.html [hunk 2]
```diff
@@ -3,28 +3,117 @@
...
+    <link rel="stylesheet" href="style.css">
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Murecho&display=swap');
+      </style>
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <!--Header with video on repeat hero!!!!!!-->
```
Comment (HWallberg) [inline code/index.html:13]: Really nice with the video!

File: code/style.css [hunk 3]
```diff
@@ -1,7 +1,295 @@
...
+.button-sub {
+  display: flex;
+  flex-direction: column;
+  align-self: flex-start;
+  padding-top: 150px;
+  padding-left: 25%;
+  margin: 20px;
+}
+
+.sub-button-style {
```
Comment (HWallberg) [inline code/style.css:59]: Made me wonder what's more accurate; to have the entire css document in chronological order, matching the html document, or collect it by type like you did here? I don't know.

File: code/style.css [hunk 4]
```diff
@@ -1,7 +1,295 @@
...
+
+.active:after {
+} */
+
+
+/*footer*/
+.footer {
+  display: block;
+  text-align: center;
+  position: fixed;
```
Comment (HWallberg) [inline code/style.css:165]: Cool feature to have the footer sticky like that, although this page didn't really need it and the footer text was kid of in the way a bit.

**Review by HWallberg — COMMENTED** [review]
This page looks great in all devices and has really clear structured code files as well. A lot if nice features was applied. For example; a video header, a button on the header, not a robot-radio button, lines around the boxes and background colors in the accordion. Fun to see! Well done!

**Pull request closed without merging**